### PR TITLE
fix recipe time and tier to adapt to CBD

### PR DESCRIPTION
**Pull request by POPlol333**
Fix chemplant enforcing as it makes no sense, and btw no one use nitrobenzene in ZPM

### CODE REVIEW

**Review by chochem — DISMISSED** [review]
that is the wrong recipe. this is just for kevlar because dreams wants it. nitrobenzene for power is in the chemplant

**Review by boubou19 — APPROVED** [review]

**Comment by POPlol333** [comment]
> that is the wrong recipe. this is just for kevlar because dreams wants it. nitrobenzene for power is in the chemplant

This is the "good recipe" as CBD isn't in chemplant and nitrobenzene is his equivalent in renewable line for is no reason for it to be in chemplant only

**Comment by POPlol333** [comment]
Not even talking about the "30s" for 5kL

**Comment by boubou19** [comment]
@chochem this is so stupid.  It's kevlar that must be changed. This is so stupid, no one is going to bother making a chemplant in HV because benzene.

**Comment by chochem** [comment]
this is nitrobenzene and meant to be EV not HV

**Comment by POPlol333** [comment]
After if you want in ANY case I'm still ok if it's not more lmao

**Review by Steelux8 — CHANGES_REQUESTED** [review]
Nitrobenzene is a Chemical Plant recipe, not an LCR recipe. The ZPM tier was decided so that there was no possibility of the LCR recipe being used for power, and only for Kevlar purposes. This ZPM placement is not meant to be reverted.

**Comment by boubou19** [comment]
> Nitrobenzene is a Chemical Plant recipe, not an LCR recipe. The ZPM tier was decided so that there was no possibility of the LCR recipe being used for power, and only for Kevlar purposes. This ZPM placement is not meant to be reverted.

https://colab.research.google.com/drive/134Mk4NiJzW-Qbb62DC_ZJymMRzL1EeQm?authuser=2#scrollTo=r_IUi480pkEL

TLDR, CBD is weaker than NitroBenzene, so NitroBenzene must be unlocked later, chemplant if it can be avoided, it will be avoided by lot of players for obvious reasons, so to keep a reason to use the chemplant recipe, it must recieve a buff (how much is the question). The fact that nitrobenzene is too strong in comparison to CBD is just a simple change to the fuel value that is beyond the scope of this PR. The fact it can be an issue for kevlar too.

**Comment by chochem** [comment]
there is a longer discussion with even some numbers in #meta-dev if some dev wants to read or chime in

**Comment by boubou19** [comment]
> there is a longer discussion with even some numbers in #meta-dev if some dev wants to read or chime in

i updated the data on the link

File: src/main/java/gregtech/loaders/postload/recipes/ChemicalRecipes.java [hunk 1]
```diff
@@ -5969,7 +5969,7 @@ public void multiblockOnly() {
                     new FluidStack(FluidRegistry.getFluid("nitrobenzene"), 5000),
                     Materials.DilutedSulfuricAcid.getFluid(3000))
                 .duration(6 * SECONDS)
-                .eut(TierEU.RECIPE_HV)
+                .eut(TierEU.RECIPE_EV)
```
Comment (chochem) [inline src/main/java/gregtech/loaders/postload/recipes/ChemicalRecipes.java:5972]: if we follow the logic in the doc this needs to be IV. aka 2x EV hatches.

**Review by chochem — CHANGES_REQUESTED** [review]

**Comment by Dream-Master** [comment]
@chochem ok now ?
